app/views/application/_footer.html.erb

```diff
@@ -33,9 +33,9 @@
     <h1>Support</h1>
 
     <h2>Hibák</h2>
-    Bármilyen hiba esetén, mint pl adminisztrációs hibák, alkalmazás hibák, kérdések, a
-    <a href="https://support.sch.bme.hu" target="_blank" rel="noreferrer">support.sch</a>-n keresztül tudod legegyszerűbben felvenni velünk a kapcsolatot,
-    vagy a <a href="mailto:kir-dev@sch.bme.hu">publikus levelezőlistánkon</a>.
+    Bármilyen hiba - mint pl adminisztrációs hibák, alkalmazás hibák, kérdések - esetén a
+    <a href="mailto:kir-dev@sch.bme.hu">publikus levelezőlistánkon</a> keresztül tudod legegyszerűbben felvenni velünk a
+    kapcsolatot.
 
     <h2>Észrevételek</h2>
```

Comment:
I think we should merge these section into Bugs section and remove the link to the form.